paasta_tools/cli/cmds/status.py

```diff
@@ -836,232 +801,87 @@ def _print_flink_status_from_job_manager(
-            f"      Taskmanager: paasta logs -a 1h -c {cluster} -s {service} -i {instance}.TASKMANAGER"
-        )
-        output.append(
-            f"      Jobmanager:  paasta logs -a 1h -c {cluster} -s {service} -i {instance}.JOBMANAGER"
-        )
-        output.append(
-            f"      Supervisor:  paasta logs -a 1h -c {cluster} -s {service} -i {instance}.SUPERVISOR"
-        )
-
-        output.append(f"{OUTPUT_HORIZONTAL_RULE}")
-
-        # Print Flink Metrics Links
-        output.append("    Flink Monitoring:")
-        output.append(
-            f"      Job Metrics: https://grafana.yelpcorp.com/d/flink-metrics/flink-job-metrics?orgId=1&var-datasource=Prometheus-flink&var-region=uswest2-{ecosystem}&var-service={service}&var-instance={instance}&var-job=All&from=now-24h&to=now"
-        )
-        output.append(
-            f"      Container Metrics: https://grafana.yelpcorp.com/d/flink-container-metrics/flink-container-metrics?orgId=1&var-datasource=Prometheus-flink&var-region=uswest2-{ecosystem}&var-service={service}&var-instance={instance}&from=now-24h&to=now"
-        )
-        output.append(
-            f"      JVM Metrics: https://grafana.yelpcorp.com/d/flink-jvm-metrics/flink-jvm-metrics?orgId=1&var-datasource=Prometheus-flink&var-region=uswest2-{ecosystem}&var-service={service}&var-instance={instance}&from=now-24h&to=now"
-        )
-
-        # Print Flink Costs Link
-        output.append(
-            f"      Flink Cost: https://app.cloudzero.com/explorer?activeCostType=invoiced_amortized_cost&partitions=costcontext%3AResource%20Summary&dateRange=Last%2030%20Days&costcontext%3AKube%20Paasta%20Cluster={cluster}&costcontext%3APaasta%20Instance={instance}&costcontext%3APaasta%20Service={service}&showRightFlyout=filters"
-        )
-
-        output.append(f"{OUTPUT_HORIZONTAL_RULE}")
-
-    # Print Flink Cluster State
-    color = PaastaColors.green if status["state"] == "running" else PaastaColors.yellow
-    output.append(f"    State: {color(status['state'].title())}")
-
-    # Print Flink Cluster Pod Info
-    pod_running_count = pod_evicted_count = pod_other_count = 0
-    # default for evicted in case where pod status is not available
-    evicted = f"{pod_evicted_count}"
-    for pod in status["pod_status"]:
-        if pod["phase"] == "Running":
-            pod_running_count += 1
-        elif pod["phase"] == "Failed" and pod["reason"] == "Evicted":
-            pod_evicted_count += 1
-        else:
-            pod_other_count += 1
-        evicted = (
-            PaastaColors.red(f"{pod_evicted_count}")
-            if pod_evicted_count > 0
-            else f"{pod_evicted_count}"
-        )
-
-    pods_total_count = pod_running_count + pod_evicted_count + pod_other_count
-    output.append(
-        "    Pods:"
-        f" {pod_running_count} running,"
-        f" {evicted} evicted,"
-        f" {pod_other_count} other,"
-        f" {pods_total_count} total"
+    # Collect instance details
```

Comment:
the function name already says this :)

i think a more valuable comment here would be something around how this will handle a null `flink_config` (that was ringing alarm bells when i first read this)